engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/marshall/ProcessMarshaller.scala

```diff
@@ -125,5 +125,8 @@ object ProcessMarshaller {
 
   val additionalNodeDataFieldsCodec: CodecJson[Option[node.UserDefinedAdditionalNodeFields]] = emptyOptionCodec
 
-  val additionalProcessFieldsCodec: CodecJson[Option[UserDefinedProcessAdditionalFields]] = emptyOptionCodec
+  val additionalProcessFieldsCodec: CodecJson[Option[UserDefinedProcessAdditionalFields]] = CodecJson.apply(
+    _ => jEmptyObject,
```

Comment:
Is encoder even needed in this case? It seems superficial right now.

Reply:
Reworked the codecs.